draft-ietf-tls-tls13.md

```diff
@@ -4866,13 +4890,20 @@ Downgrade protection.
 should be the same as if the peers had been communicating in the
 absence of an attack (See {{BBFKZG16}}; defns 8 and 9}).
 
-Forward secret
+Forward secret with respect to long-term keys
 : If the long-term keying material (in this case the signature keys in certificate-based
 authentication modes or the external/resumption PSK in PSK with (EC)DHE modes) are compromised after
 the handshake is complete, this does not compromise the security of the
 session key (See {{DOW92}}).  The forward secrecy property is not satisfied
```

Comment:
I'm not sure about that, the session key is defined and used all over the place. Also you could think that forward secrecy also affects the handshake secret which includes the master secret but the reverse is not true.